costmap_2d/include/costmap_2d/observation_buffer.h

```diff
@@ -70,15 +70,16 @@ class ObservationBuffer
    * @param  max_obstacle_height The minimum height of a hitpoint to be considered legal
    * @param  obstacle_range The range to which the sensor should be trusted for inserting obstacles
    * @param  raytrace_range The range to which the sensor should be trusted for raytracing to clear out space
+   * @param  start_clearing_range The range to wich the sensor start to be trusted for clearing out space.
    * @param  tf A reference to a TransformListener
    * @param  global_frame The frame to transform PointClouds into
    * @param  sensor_frame The frame of the origin of the sensor, can be left blank to be read from the messages
    * @param  tf_tolerance The amount of time to wait for a transform to be available when setting a new global frame
    */
   ObservationBuffer(std::string topic_name, double observation_keep_time, double expected_update_rate,
                     double min_obstacle_height, double max_obstacle_height, double obstacle_range,
-                    double raytrace_range, tf::TransformListener& tf, std::string global_frame,
-                    std::string sensor_frame, double tf_tolerance);
+                    double raytrace_range, double start_clearing_range, tf::TransformListener& tf,
+                    std::string global_frame, std::string sensor_frame, double tf_tolerance);
```

Comment:
same comments as we did for Observation -- leave this constructer, default min_raytrace_range to 0.0, deprecate constructor. Add new constructor with min before max. Since we'll rename raytrace range to max_raytrace_range, let's also make sure the doxygen gets updated